src/components/app/data/queries/utils.ts

```diff
@@ -93,6 +93,7 @@ export async function getEnterpriseLearnerQueryData({
     enterpriseFeatures: {},
     staffEnterpriseCustomer: null,
     shouldUpdateActiveEnterpriseCustomerUser: false,
+    hasBnrEnabledPolicy: false, // Added missing property
```

Comment:
Nit: Please remove the following comment from all places.
// Added missing property